clustering-map/src/App.js

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+  client,
+  useConfig,
+  useElementColumns,
+  useElementData,
+} from "@sigmacomputing/plugin";
+
+import './App.css';
+import Map from './components/Map';
+
+client.config.configureEditorPanel([
+  { name: "source", type: "element" },
+  { name: "Latitude", type: "column", source: "source", allowMultiple: false },
```

Comment:
You could limit the column types this config accepts with `allowedTypes`

Reply:
Thanks for the suggestion, added!